server/controllers/user_controller.js

```diff
@@ -94,6 +87,40 @@ class UserController {
       });
     }
   }
-}
+  static changeToMentor = async (req, res) => {
+
+      const id = req.params.id;
+      notNumber(id, res)
+      const mentor = await this.model().select('*', 'id=$1', [id]);
```

Comment:
You a try/catch here, that way you won't need the first if statement `if (!mentor[0])` and the response will be returned in the catch closure.
something like this:
```try {
  const mentor = await this.model().select('*', 'id=$1', [id]);
  ....
}catch {
  return res.status(status.NOT_FOUND).send({
          status: status.NOT_FOUND,
          error: `User with this id ${id} does not exist`,
        });
}```